FirebaseMessaging.podspec

```diff
@@ -63,4 +63,12 @@ device, and it is completely free.
     }
     unit_tests.dependency 'OCMock'
   end
+
+  s.test_spec 'sample' do |build_test|
```

Comment:
Not necessary to do it this way when you already have a Podfile. You could also follow a pattern like https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/symbolcollision.yml

Reply:
Done.